src/config.php.dist

```diff
@@ -1,8 +1,8 @@
 <?php
 define('CONFIG_VERSION', "7");
 define('LOG_FILE', "/tmp/msqur.log"); //Ew
-define('DB_HOST', "localhost");
-define('DB_USERNAME', "msqur");
+define('DB_HOST', "127.0.0.1");
```

Comment:
I don't see a need to change these are default values, any of your machine's changes should be in `config.php`.